Remotion/Web/Development.WebTesting/BrowserSession/Chrome/ChromeBrowserSession.cs

```diff
@@ -31,18 +29,16 @@ namespace Remotion.Web.Development.WebTesting.BrowserSession.Chrome
   /// </summary>
   public class ChromeBrowserSession : BrowserSessionBase<IChromeConfiguration>
   {
-    private static readonly ILog s_log = LogManager.GetLogger (typeof (ChromeBrowserSession));
-
-    private readonly string _userDirectory;
+    private readonly IBrowserSessionCleanupStrategy[] _cleanUpStrategies;
 
     public ChromeBrowserSession (
         [NotNull] Coypu.BrowserSession value,
         [NotNull] IChromeConfiguration configuration,
         int driverProcessID,
-        [CanBeNull] string userDirectory)
+        [ItemNotNull] IBrowserSessionCleanupStrategy[] cleanUpStrategies = default)
```

Comment:
If I'm not mistaken the default for an array is null, maybe specify `null` instead of `default`?